.github/workflows/lint.yml

```
@@ -0,0 +1,64 @@
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install SwiftLint
+        env:
+          HOMEBREW_NO_AUTO_UPDATE: 1
+        run: brew install swiftlint
+
+      - name: Run SwiftLint
+        working-directory: kmp_client/iosApp
+        run: swiftlint lint --config .swiftlint.yml --quiet
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)